devops/providers/azure-devops/templates/infrastructure/azure-pipeline-release-stage.yml

```diff
@@ -1,11 +1,13 @@
 parameters:
+  alwaysRun: 'false'
   environment: ''
   configurationMatrix: []
 
 jobs:
 
 - job: TemplateChangeDetection
   displayName: Determine CD Targets to Run
+  condition: ne(${{ parameters.alwaysRun }}, 'true')
```

Comment:
Similar comment as above. Does line 25 take care of this? If always run is false, then our existing change detection won't ever be included in the template